### PR TITLE
Disallow Nan, +-Infinity measurements

### PR DESCRIPTION
**Pull request by abaranch**
#102 
Pull request to master because we want to include this bug in the stable version.

### CODE REVIEW

**Comment by abaranch** [comment]
Build is failing because CodeCode is not in master yet.

File: src/Core/Managed/Shared/Extensibility/Implementation/Property.cs [hunk 1]
```diff
@@ -141,8 +141,15 @@ public static void SanitizeMeasurements(this IDictionary<string, double> diction
 
                     string sanitizedKey = SanitizeKey(entry.Key, dictionary);
 
+                    // Disallow Nan and Infinity since Breeze does not accept it
+                    double sanitizeValue = entry.Value;
```
Comment (SergeyKanzhelev) [inline src/Core/Managed/Shared/Extensibility/Implementation/Property.cs]: do we have any other double values anywhere? I believe Metric may have it and Measurements colleciton. Worth checking maybe we have anything else that potentially can become NaN

Comment (abaranch) [inline src/Core/Managed/Shared/Extensibility/Implementation/Property.cs]: We have it also in Metric and in Sampling. I updated the first one and I do mot think we need to update the second.

**Comment by SergeyKanzhelev** [comment]
:shipit: I'd suggest to sanitize `Sampling` anyway. It may already be sanitized by `>0` and '<100' though

**Comment by abaranch** [comment]
We already check sampling: https://github.com/Microsoft/ApplicationInsights-dotnet/blob/master/src/Core/Managed/Shared/Extensibility/Implementation/Telemetry.cs#L17

**Pull request merged**